src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/SupplementaryEvidenceMapperTest.java

```diff
@@ -96,14 +96,14 @@ void should_not_add_document_from_envelope_if_document_with_the_same_control_num
         // given
         List<Document> existingDocs =
             asList(
-                new Document("a.pdf", "AAA", "type_a", "subtype_a", now().plusSeconds(1), "http://localhost/a.pdf"),
-                new Document("b.pdf", "BBB", "type_b", "subtype_b", now().plusSeconds(2), "http://localhost/b.pdf")
+                new Document("a.pdf", "AAA", "type_a", "subtype_a", now().plusSeconds(1), "http://localhost/uuida", "uuida"),
```

Comment:
It would be nice to use proper url format with uuid

Reply:
Updated all test data

Reply:
@nitinprabhu I updated the test data but I don't think it's readable. I think it's better to leave the test data as it was before, which was easy to understand what the test is about.

Reply:
In that case how would anyone looking at the code can tell that how the document url looks like ? There is inconsistency in the test data being used across tests. It would best to keep it consistent and also helps anyone looking at the code to figure out the format of the url.

Reply:
I updated the test data as you suggested. But I don't think we need a valid uuid in the unit tests. In the functional tests, we are using actual envelope data. but these are just unit tests where we have dummy data. I think 
                **`new Document("b.pdf", "BBB", "type_b", "subtype_b", now().plusSeconds(2), "http://localhost/uuidb", "uuidb")`**
is easy to understand the test, compare to the below:
                `new Document("b.pdf", "BBB", "type_b", "subtype_b", now().plusSeconds(2), "https://example.gov.uk/f8a5f699-3816-4298-9365-ecb5b634c8f6", "f8a5f699-3816-4298-9365-ecb5b634c8f6")`

Reply:
Yeah makes sense for unit test 👍